api/billingd/service/service.go

```diff
@@ -505,11 +509,19 @@ func (s *Service) createCustomer(
 	}
 	log.Debugf("created customer %s with id %s", doc.Key, doc.CustomerID)
 
-	s.analytics.Identify(doc.Key, doc.AccountType, false, doc.Email, map[string]interface{}{
-		"parent_key":  doc.ParentKey,
-		"customer_id": doc.CustomerID,
-		"username":    params.Username,
-	})
+	if err := s.analytics.Identify(
+		ctx,
+		doc.Key,
+		doc.AccountType.Pb(),
+		analytics.WithEmail(doc.Email),
+		analytics.WithProperties(map[string]interface{}{
+			"parent_key":  doc.ParentKey,
+			"customer_id": doc.CustomerID,
+			"username":    params.Username,
+		}),
+	); err != nil {
+		log.Errorf("calling analytics identify: %v", err)
+	}
```

Comment:
In the case of the analytics client, you don't have to check if it's `nil` or not because the API implementations do a noop if it is nil. I'll comment on that code.

Reply:
good stuff